src/pages/About/Step.tsx

```diff
@@ -31,18 +32,22 @@ const Step = ({
   title,
   description,
   isLast,
+  expanded,
+  onClick,
 }: {
   index: number
   title: string
   description: string
   isLast?: boolean
+  onClick: () => void
+  expanded?: boolean
 }) => {
   return (
-    <StyledStep isLast={isLast}>
+    <StyledStep isLast={isLast} onClick={onClick}>
```

Comment:
Can `isLast` be moved to a css selector?

Reply:
good point yes